styleguide/template.md

```diff
@@ -222,44 +222,34 @@ You can use a [Markdown table generator tool](http://www.tablesgenerator.com/mar
 The best way to include code is to include snippets from a working sample. Create your
 sample following the instructions in the [contributing guide](../CONTRIBUTING.md#contributing-to-samples).
 
```

Comment:
On line 39, for the **ms.devlang** tag, do you want to add PowerShell?

Reply:
yes, PowerShell should be added (IMHO!) :-)